pkg/backend/apply.go

```diff
@@ -75,6 +75,7 @@ type response string
 
 const (
 	yes     response = "yes"
+	yesPlan response = "[experimental] yes, using Update Plans (https://www.pulumi.com/docs/intro/concepts/plans)"
```

Comment:
Nit: It'd be nice if the URL was shorter to keep the line shorter. What about the URL that Luke suggested, https://pulumi.com/updateplans, that redirects to https://www.pulumi.com/docs/intro/concepts/plans, similar to how we have https://pulumi.com/start which redirects to https://www.pulumi.com/docs/get-started ?

Reply:
If we can get that short url set up then sure.

Reply:
Added a comment on it at https://github.com/pulumi/pulumi-hugo/pull/2230/files#r1022698093